SDK/AppCenterPush/Microsoft.AppCenter.Push.UWP/Push.cs

```diff
@@ -141,6 +141,53 @@ private void OnPushNotificationReceivedHandler(PushNotificationChannel sender, W
             }
         }
 
+        private async Task<PushNotificationChannel> CreatePushNotificationChannelAsync()
+        {
+            PushNotificationChannel channel = null;
+            try
+            {
+                // If this isn't the first time after installation, the notification channel is created successfully even without network.
+                channel = await new WindowsPushNotificationChannelManager().CreatePushNotificationChannelForApplicationAsync()
+                    .AsTask().ConfigureAwait(false);
+            }
+            catch (Exception exception)
```

Comment:
Is it necessary to catch any exception type, or can you make this more specific?

Reply:
Yes, `System.Exception` should be caught.
Sample one:
```C#
System.Exception occurred
  HResult=0x800704C6
  Message=The network is not present or not started. (Exception from HRESULT: 0x800704C6)
  Source=<Cannot evaluate the exception source>
  StackTrace:
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at Microsoft.AppCenter.Push.Push.<CreatePushNotificationChannelAsync>d__7.MoveNext() in SDK\AppCenterPush\Microsoft.AppCenter.Push.UWP\Push.cs:line 150
```